docs/deployment-examples/database-deployments/sql-server/dacpac.md

```diff
@@ -0,0 +1,299 @@
+---
+title: SQL Server DACPAC deployment
+description: How to do database deployments with DACPAC.
+position: 20
+---
+
+
+Starting with SQL Server 2008, Microsoft introduced a new project type called Database Projects.  These projects use the [state-based approach](https://octopus.com/blog/sql-server-deployment-options-for-octopus-deploy) to applying changes to your database.  Initially, Database Projects were not available as part of the initial Visual Studio install and had to be downloaded separately. This download was referred to as SQL Server Data Tools (SSDT) and included project types for Database projects, SQL Server Reporting Services (SSRS) projects, and SQL Server Integration Services (SSIS) projects.  Modern versions of Visual Studio have SSDT available to choose when installing or modifying an existing installation.
+
+## Installing SSDT for Visual Studio
+For earlier versions of Visual Studio such as 2015 and below, installing the SSDT was a matter of locating the download for your version of Visual Studio.  Microsoft has provided a convenient way of finding the appropriate download on [this page](https://docs.microsoft.com/en-us/sql/ssdt/previous-releases-of-sql-server-data-tools-ssdt-and-ssdt-bi?view=sql-server-ver15).
```

Comment:
For the installation of SSDT, I'd prefer we link to an authoritative source for that information, as it could change and we wouldn't (so the doc won't be updated), if we link, the user will still be able to complete the task, and if we use OctoPurl to create the link, we'll be notified if the link ever disappears, at which point we can update the link.

Reply:
@twerthi, could you take a look and update this?

Reply:
@twerthi nudge :)

Reply:
Sleeping at the wheel ;)  I'll take a look.